<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mandalaman</class>
 <widget class="QMainWindow" name="Mandalaman">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mandalaman</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="upperToolBar" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <underline>false</underline>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="sliceLabel">
         <property name="text">
          <string>Slice</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliceSlider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sliceSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mirrorCheckBox">
         <property name="text">
          <string>Mirror</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="resolutionDropDownList">
         <item>
          <property name="text">
           <string>500x500</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>400x400</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearBtn">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Grid</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="gridOpacitySlider">
         <property name="sliderPosition">
          <number>99</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="futurCanevasLayout" native="true">
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <widget class="painter" name="paintingZone" native="true">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>0</y>
         <width>500</width>
         <height>500</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>500</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lineWidthLabel">
         <property name="text">
          <string>Line width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="lineWidthSlider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="pageStep">
          <number>3</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colorBtn">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rainbowColorBtn">
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../rainbowBtnImg.png</normaloff>../../rainbowBtnImg.png</iconset>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_as"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>painter</class>
   <extends>QWidget</extends>
   <header>painter.h</header>
   <container>1</container>
   <slots>
    <slot>penWidth(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sliceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliceSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliceSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliceSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>penWidth(int)</slot>
 </slots>
</ui>
